ticketguardian/order/order.py

```diff
@@ -14,11 +15,15 @@
 class Order(
         ListResourceMixin,
         RetrieveResourceMixin,
+        PatchResourceMixin,
         PostResourceMixin,
         PutResourceMixin, ):
 
     resource = "orders"
 
+    def __init__(self):
+        super(Order, self).__init__()
```

Comment:
Why is this necessary? Wouldn't the super method also be called if you just removed the method entirely?

Reply:
There are a few instances of this all over, can I assume this is a python2 compatibility thing?